### PR TITLE
Fixed opacity of the greetings card

### LINKED ISSUE #4: FEATURE: Farmer greetings message UI improvement
### Have you updated the title?

- [X] I have updated the issue's title.

### As a...

user

### I can...

not be overwhelmed by the large font of the greeting text and also appreciate the artistic design of the message

### So that...

I can have an improved user experience

### Discussion

1. Having greeting messages in home page and common room page is a great idea in general, but currently the font size is too big and it's just a plain text with black background color.
2. Make the Greeting text slightly smaller, and maybe add movement, hover effect, animation, or gradient background color to make it more stand out.
3. The design is entirely up to the students.

### Acceptance Criteria

- [ ] Home page greeting text should have smaller font size than before as well as more sophisticated CSS effects.
- [ ] Play page greeting text should have smaller font size than before as well as more sophisticated CSS effects.

### Implementation Todos

Frontend:
- [ ] Home page greeting text should have smaller font size than before as well as more sophisticated CSS effects.
- [ ] Play page greeting text should have smaller font size than before as well as more sophisticated CSS effects.

Testing:
- [ ] Frontend: greeting text should have smaller font size and more sophisticated css effects, and students should write unit tests check those attributes.

### PR DESCRIPTION
**Pull request by AngelG261**
## Overview

In this PR, I fixed the opacity of the greeting card so that it matched the default cards (additional formatting).

## Screenshots (Optional)
Before:
image

After:
image

Closes #4

### CODE REVIEW

**Review by haroldmo — DISMISSED** [review]
The storybook looks good. The opacity matches the default greeting cards.

**Review by Yashasvi-Vangala — DISMISSED** [review]
LGTM!
Cannot see the transparency on local host so I deployed the branch on: [https://5pm-1-happycows-qa.dokku-01.cs.ucsb.edu](https://5pm-1-happycows-qa.dokku-01.cs.ucsb.edu)

File: frontend/src/main/components/Commons/CommonsPlay.js [hunk 1]
```diff
@@ -10,8 +10,9 @@ export default function CommonsPlay({ currentUser }) {
   // Stryker restore all
 
   return (
+      // Stryker disable all : TODO: restructure this code to avoid the need for this disable
```
Comment (pconrad) [inline frontend/src/main/components/Commons/CommonsPlay.js]: Nah, we dont need the cheat code here.  This is straightforward to test.

Please remove this and fix the test gaps.

Here's an example of how to check for a CSS property, from [`frontend/src/tests/pages/PlayPage.test.js`](https://github.com/ucsb-cs156-f23/proj-happycows-f23-5pm-1/blob/main/frontend/src/tests/pages/PlayPage.test.js)

```js
```suggestion

```
 var div = screen.getByTestId("playpage-div");
        expect(div).toHaveAttribute("style", expect.stringContaining("background-size: cover; background-image: url(PlayPageBackground.png);"));
```

**Review by pconrad — CHANGES_REQUESTED** [review]
LGTM except for the stryker exception.

Please remove this and add the tests that are needed.  This is straightforward to test for and there are already examples in the code base.  See the code review for a hint.

File: frontend/src/main/components/Commons/CommonsPlay.js (hunk 1 above)
Comment (Yashasvi-Vangala) [reply frontend/src/main/components/Commons/CommonsPlay.js]: Fixed it with the latest commits

**Review by haroldmo — APPROVED** [review]
The extra tests were added and the stryker disable all was removed. This looks good to me. All tests passed as well locally.

**Review by pconrad — APPROVED** [review]
LGTM!

**Pull request merged**